packages/integration-karma/test/shadow-dom/ShadowRoot-properties/ShadowRoot.delegatesFocus.spec.js

```diff
@@ -3,24 +3,24 @@ import { createElement } from 'lwc';
 
 describe('ShadowRoot.delegatesFocus', () => {
     // TODO [#985]: delegatedFocus is only implemented the native ShadowRoot by Blink
```

Comment:
I think you can dedupe #2126 and #985. `delegatesFocus` is now implemented by Webkit, Gecko is the only one missing.